Backend build (publish) fails with warnings as errors

There are additional warnings that are generated when the backend is published.  Since #342 these are treated as errors so the build will fail:
```
Controllers/UserEditController.cs(56,42): error CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread. [/home/grady/projects/sil/TheCombine/Backend/BackendFramework.csproj]
Controllers/FrontierController.cs(51,42): error CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread. [/home/grady/projects/sil/TheCombine/Backend/BackendFramework.csproj]
Controllers/FrontierController.cs(77,42): error CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread. [/home/grady/projects/sil/TheCombine/Backend/BackendFramework.csproj]
```
The backend is being built with the commands:
```bash
cd Backend
dotnet publish -c Release
```

--- a/Backend/Controllers/FrontierController.cs
+++ b/Backend/Controllers/FrontierController.cs
@@ -48,7 +48,10 @@ namespace BackendFramework.Controllers
         /// <remarks> POST: v1/projects/{projectId}/words/frontier </remarks>
         /// <returns> Id of word created </returns>
         [HttpPost]
+        // TODO: Remove this warning suppression when the function is implemented for release mode.
+#pragma warning disable 1998
         public async Task<IActionResult> PostFrontier(string projectId, [FromBody]Word word)
+#pragma warning restore 1998
         {
 #if DEBUG
             if (!_permissionService.HasProjectPermission(Permission.WordEntry, HttpContext))
@@ -74,7 +77,10 @@ namespace BackendFramework.Controllers
         /// <summary> Deletes all words in a project's frontier </summary>
         /// <remarks> DELETE: v1/projects/{projectId}/words/frontier/{wordId} </remarks>
         [HttpDelete("{wordId}")]
+        // TODO: Remove this warning suppression when the function is implemented for release mode.
+#pragma warning disable 1998
         public async Task<IActionResult> DeleteFrontier(string projectId, string wordId)
+#pragma warning restore 1998
         {
 #if DEBUG
             if (!_permissionService.HasProjectPermission(Permission.WordEntry, HttpContext))

--- a/Backend/Controllers/UserEditController.cs
+++ b/Backend/Controllers/UserEditController.cs
@@ -53,7 +53,10 @@ namespace BackendFramework.Controllers
         /// <remarks> DELETE: v1/projects/{projectId}/useredits </remarks>
         /// <returns> true: if success, false: if there were no projects </returns>
         [HttpDelete]
+        // TODO: Remove this warning suppression when the function is implemented for release mode.
+#pragma warning disable 1998
         public async Task<IActionResult> Delete(string projectId)
+#pragma warning restore 1998
         {
 #if DEBUG
             if (!_permissionService.HasProjectPermission(Permission.DatabaseAdmin, HttpContext))

